doc/interfaces/denoise.rst

```diff
@@ -0,0 +1,223 @@
+.. _denoise:
+
+=========
+Denoising
+=========
+
+This tutorial walks through the steps to denoise diffusion-weighted MR images using DIPY.
+Multiple denoising methods are available in DIPY.
+
+You can try these methods using your own data; we will be using the data in DIPY.
+You can check how to `fetch the DIPY data <https://github.com/dipy/dipy/blob/master/doc/interfaces/data_fetch.rst>`_.
+
+-------------------------
+Denoising using Local PCA
+-------------------------
+
+PCA-based denoising algorithms are effective denoising methods because they explore 
+the redundancy of the multi-dimensional information of diffusion-weighted datasets.
+
+The algorithm to perform the PCA-based denoising involves the following steps:
+
+* First, we estimate the local noise variance at each voxel.
+
+* Then, we apply PCA in local patches around each voxel over the gradient directions.
+
+* Finally, we threshold the eigenvalues based on the local estimate of sigma and then do a PCA reconstruction
+
+We are using ``stanford_hardi`` dataset. You can also use your own data.
+
+The workflow for the LPCA denoising requires the paths to the diffusion input file,
+as well as the b-values and b-vectors files.
+
+You may want to create an output directory to save the denoised data::
+
+    mkdir denoise_lpca_output
+
+Run the following command::
+
+    dipy_denoise_lpca data/stanford_hardi/HARDI150.nii.gz data/stanford_hardi/HARDI150.bval data/stanford_hardi/HARDI150.bvec --out_dir "denoise_lpca_output"
+
+This command will denoise the diffusion image and save it to the directory
+``denoise_lpca_output``.
+
+In case, you did not specify ``out_dir``, the denoised diffusion image will 
+be saved to a file named ``dwi_lpca.nii.gz`` by default, located in the 
+input directory also by default.
+
+.. |image1| image:: https://github.com/dipy/dipy_data/blob/master/stanford_hardi_original.png  
```

Comment:
`https://github.com/dipy/dipy_data/blob/master/stanford_hardi_original.png?raw=true` instead

Reply:
Addressed in 12ceeef